bundle_util.ts

```diff
@@ -36,3 +61,33 @@ export function setImportMap(importMap: string) {
 export function getImportMap() {
   return _importMap;
 }
+
+let _tsconfig: string | undefined;
+
+export function setTsconfig(tsconfig: string) {
+  _tsconfig = tsconfig;
+}
+
+export async function getTsconfig() {
+  if (!_tsconfig) {
+    switch (true) {
+      case await exists("./deno.json", {
+        isReadable: true,
+        isDirectory: false,
+      }):
+        return "./deno.json";
+      case await exists("./denoc.json", {
```

Comment:
This should be `deno.jsonc` and it can be have comments. (`JSON.parse` will cause parse error in that case)

Reply:
Then I will add another module to `deps.ts` ([jsonc](https://deno.land/std@0.193.0/jsonc/mod.ts))